Jenkinsfile

```diff
@@ -20,7 +20,8 @@ pipeline {
                 sh 'build/run make -j\$(nproc) test'
             }
         }
-        stage('Integration Tests') {
+
+        stage('Smoke Test - k8s 1.6'){
```

Comment:
I was thinking we would run the smoke test first, and then run a full regression on the whole matrix.